.golangci.yml

```diff
@@ -24,6 +26,7 @@ run:
 
 linters:
   enable:
+    - gofmt
```

Comment:
I think we don't even need this. Looking at `golangci-lint linters | grep goimports`:
```
goimports: Goimports does everything that gofmt does. Additionally it checks unused imports [fast: true, auto-fix: true]
```

Reply:
not exactly correct.

goimports version of golangci doesn't do simplification.

Reply:
here is the configuration: https://github.com/golangci/golangci-lint/blob/master/pkg/config/config.go#L139

maybe we should suggest adding it then goimports can be used solely.

Reply:
Mm.. looks like goimports does no simplifications: https://github.com/golang/go/issues/21476

Reply:
There was a long standing demand, I thought it was already added but it seems it's still missing.